express-hello-world/server.js

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+
+const app = express();
+
+app.use((req, res) => {
+  console.log(req.method);
+  console.log(res.send('Hello, world!!'));
+});
+
+app.listen(8000, () => {
```

Comment:
From now on, please use port 8080. All the subsequent exercises where starting code is provided are set up to use that port.